test/fixtures/rules/std_rule_cases/L027.yml

```diff
@@ -61,12 +61,34 @@ test_allow_date_parts_as_function_parameter_snowflake:
     core:
       dialect: snowflake
 
-test_ignore_value_table_functions:
+test_ignore_value_table_functions_when_counting_tables:
```

Comment:
This, despite the slightly weird diff, is just the old test `test_ignore_value_table_functions` updated to
- Have a new name for the specific case it is testing
- Have make the reference to a column from `a` unqualified (to verify that this still passes lint, now that `_t_start` is ignored)